velox/vector/arrow/Bridge.cpp

```diff
@@ -644,6 +663,26 @@ void exportToArrow(const VectorPtr& vec, ArrowSchema& arrowSchema) {
     arrowSchema.dictionary = bridgeHolder->dictionary.get();
     exportToArrow(vec->valueVector(), *arrowSchema.dictionary);
 
+  } else if (vec->encoding() == VectorEncoding::Simple::CONSTANT) {
```

Comment:
It helps to add a link to the spec https://arrow.apache.org/docs/format/Columnar.html#run-end-encoded-layout